ui/app/store/actions.js

```diff
@@ -322,7 +322,6 @@ var actions = {
   setAutoLogoutTimeLimit,
 
   // Migration of users to new UI
-  setCompletedUiMigration,
   completeUiMigration,
   COMPLETE_UI_MIGRATION: 'COMPLETE_UI_MIGRATION',
```

Comment:
Is this action still useful?

Reply:
Nope, good catch! I had thought I'd removed that.